products/ansible.md

```diff
@@ -13,23 +13,28 @@ iconSlug: ansible
 auto:
   git: https://github.com/ansible-community/ansible-build-data.git
 releases:
-  - releaseCycle: "5"
+-   releaseCycle: "5"
     release: 2021-11-30
     eol: false
     latest: "5.7.1"
-  - releaseCycle: "4"
-    release: 2021-05-11
+    latestReleaseDate: 2022-05-04
+-   releaseCycle: "4"
+    release: 2021-05-18
     eol: true
     latest: "4.10.0"
-  - releaseCycle: "3"
-    release: 2021-02-09
+    latestReleaseDate: 2021-12-14
+-   releaseCycle: "3"
+    release: 2021-02-18
```

Comment:
Is accurate, https://github.com/endoflife-date/release-data/blob/58c73317232e7012d8abac04478dd261b705e051/releases/git/ansible.json#L10

https://github.com/ansible-community/ansible-build-data/releases/tag/3.0.0